controllers/spacebindingcleanup/spacebinding_cleanup_controller.go

```diff
@@ -26,6 +27,12 @@ import (
 const norequeue = 0 * time.Second
 const requeueDelay = 10 * time.Second
 
+type PublicViewerConfig interface {
+	Enabled() bool
+}
+
+type PublicViewerConfigProvider func(context.Context, client.Client) PublicViewerConfig
+
```

Comment:
as mentioned we could avoid those if we read the configuration in the reconcile as we do for other controllers.

Reply:
done in 022a04807f71b648f7564ca41bc0e9f4e16b9d29